src/edu/csus/ecs/pc2/clics/API202306/ResourceConfig202306.java

```diff
@@ -0,0 +1,126 @@
+
+        // CLICS Contest API services are collective -- either all enabled or all disabled (and default to enabled if unspecified)
+        if (wsPropUtilities.getBooleanProperty(WebServerPropertyUtils.CLICS_CONTEST_API_SERVICES_ENABLED_KEY, true)) {
+
+            resConfig.register(new ContestService(getContest(), getController()));
+            showMessage("Starting /contests web service");
+            resConfig.register(new AccessService(getContest(), getController()));
+            showStartingMessage("access web service");
+            resConfig.register(new AccountService(getContest(), getController()));
+            showStartingMessage("accounts web service");
+            resConfig.register(new ScoreboardService(getContest(), getController()));
+            showStartingMessage("scoreboard web service");
+            resConfig.register(new LanguageService(getContest(), getController()));
+            showStartingMessage("languages web service");
+            resConfig.register(new TeamService(getContest(), getController()));
+            showStartingMessage("teams web service");
+            resConfig.register(new GroupService(getContest(), getController()));
+            showStartingMessage("groups web service");
+            resConfig.register(new OrganizationService(getContest(), getController()));
+            showStartingMessage("organizations web service");
+            resConfig.register(new JudgementTypeService(getContest(), getController()));
+            showStartingMessage("judgement-types web service");
+            resConfig.register(new ClarificationService(getContest(), getController()));
+            showStartingMessage("clarifications web service");
+            resConfig.register(new SubmissionService(getContest(), getController()));
+            showStartingMessage("submissions web service");
+            resConfig.register(new ProblemService(getContest(), getController()));
+            showStartingMessage("problems web service");
+            resConfig.register(new JudgementService(getContest(), getController()));
+            showStartingMessage("judgements web service");
+            resConfig.register(new RunService(getContest(), getController()));
+            showStartingMessage("runs web service");
+            resConfig.register(new EventFeedService(getContest(), getController()));
+            showStartingMessage("event-feed web service");
+            resConfig.register(new StateService(getContest(), getController()));
+            showStartingMessage("state web service");
+            resConfig.register(new AwardService(getContest(), getController()));
+            showStartingMessage("awards web service");
+            resConfig.register(new VersionService(getContest(), getController()));
+            showMessage("Starting / endpoint for version web service");
+
+        }
+
+        return resConfig;
+    }
+
+    // Maybe these should be in a base class that all API's inherit?
```

Comment:
Sure seems like it, since all the following methods are going to be duplicated in each API version...